databroker/broker.py

```diff
@@ -1068,10 +1074,10 @@ def inner(stream):
 
 def header_io(db_in, db_out):
     def outer(f):
-        def inner(header):
+        def inner(headers):
```

Comment:
Actually there is no need for this. `restream` accepts a list of headers and returns one long stream, so this use case should work as is.